### DISCUSSION TITLE
Welcome to MMM-01ZM Discussions!

### DISCUSSION THREAD

**Discussion by rubinho101** [rubinho101/MMM-01ZM#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ludofrench** [comment]
Hi rubinho, the module don't work for me. I have 2 step with a problem.
postinstaller.sh executable don't work ( impossible access to img/postinstaller.sh, no file or folder.)
git clone ......./mitemp same reason. The destination path "mitemp" it already exists and is not an empty directory.
If you have an idea for my problem.
Thanks you so much.
Ludovic

**Reply by rubinho101** [reply]
Hi @ludofrench can you post the console log? did you make the postinstaller.sh executable before `npm install` in the MMM-01ZM folder?
Regards

**Reply by ludofrench** [reply]
Thanks for responding, i have this message on console:

pi@raspberrypi:~ $ cd /home/pi/MagicMirror/modules/MMM-01ZM
pi@raspberrypi:~/MagicMirror/modules/MMM-01ZM $ chmod +x /home/pi/MMM-01ZM/img/postinstaller.sh
chmod: impossible d'accéder à '/home/pi/MMM-01ZM/img/postinstaller.sh': Aucun fichier ou dossier de ce type

**Reply by rubinho101** [reply]
there is a slight typo in line 2: it should be `chmod +x /home/pi/MagicMirror/modules/MMM-01ZM/img/postinstaller.sh`

**Reply by ludofrench** [reply]
It works :+1:  
other problem with hassiweb/mitemp. doesn't work:

pi@raspberrypi:~/MagicMirror/modules/MMM-01ZM $ pip3 install bluepy
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Requirement already satisfied: bluepy in /home/pi/.local/lib/python3.7/site-packages (1.3.0)
pi@raspberrypi:~/MagicMirror/modules/MMM-01ZM $ pip3 install btlewrap
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Requirement already satisfied: btlewrap in /home/pi/.local/lib/python3.7/site-packages (0.1.0)
pi@raspberrypi:~/MagicMirror/modules/MMM-01ZM $ git clone https://github.com/hassiweb/mitemp
fatal: le chemin de destination 'mitemp' existe déjà et n'est pas un répertoire vide.

I clone this one in /MMM-01ZM/mitemp but don't work too.

**Reply by rubinho101** [reply]
are you trying to install it manually or via the postinstaller.sh? I dont speak french fluently, but it looks like you got MMM-01ZM/mitemp already cloned from github. Next step would be to move the parseBLE.py into the mitemp folder and change the argument in mitemp_bt_poller.py as written in the readme.
